### PR TITLE
[meteostick] Minor corrections to README.md

### PR DESCRIPTION
**Pull request by kermitd**
Add New Zealand to channel list, correction to things example

### CODE REVIEW

**Comment by martinvw** [comment]
You should sign off the commit see http://docs.openhab.org/developers/contributing/contributing.html#sign-your-work instead of the pr message

File: addons/binding/org.openhab.binding.meteostick/README.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ Next add the sensor and configure the channel number.
...
 ### meteostick_bridge Configuration Options
 
-| Option | Description                                                                |
-|--------|----------------------------------------------------------------------------|
-| port   | Sets the serial port to be used for the stick                              |
-| mode   | Sets the operating mode 0 = USA, 1 = Europe, 2 = Australia. 3 = FineOffset |
+| Option | Description                                                                                |
+|--------|--------------------------------------------------------------------------------------------|
+| port   | Sets the serial port to be used for the stick                                              |
+| mode   | Sets the operating mode 0 = USA, 1 = Europe, 2 = Australia, 3 = FineOffset, 5= New Zealand |
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.meteostick/README.md]: What is 4?

**Comment by cdjackson** [comment]
Please also add this to the thing definition file, otherwise it will only be usable in the DSL.

The current meteostick documentation doesn't seem to include this - I'm assuming that you've tested it?

File: addons/binding/org.openhab.binding.meteostick/README.md (hunk 1 above)
Comment (kermitd) [inline addons/binding/org.openhab.binding.meteostick/README.md]: Good point.  Have found 4 in the documentation and updated accordingly

**Comment by kermitd** [comment]
Have updated thing definition as well (I think?).  Yes, documentation doesn't include NZ band but I have confirmed with the manufacturer and tested.

**Comment by cdjackson** [comment]
Thanks - looks good - thanks. 

As @martinvw mentioned above, you should sign the commits - or squash them and sign the squashed commit.

**Comment by martinvw** [comment]
Can you squash your commits and change commit message to include a `Signed-off-by` (to certify that you wrote it or otherwise have the right to pass it on as an open-source), see http://docs.openhab.org/developers/contributing/contributing.html#sign-your-work

For squashing see:
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by kermitd** [comment]
Sorry guys, I can't figure out how to either edit my commit messages or squash the commits on GitHub.   First time using git so not sure how to move from editing on GitHub to using on command line in order to do either of these.

**Comment by martinvw** [comment]
@kaikreuzer can we see this as a minor change or is there another way to comply to the contributor guidelines?

**Comment by martinvw** [comment]
> Sorry guys, I can't figure out how to either edit my commit messages or squash the commits on GitHub. First time using git so not sure how to move from editing on GitHub to using on command line in order to do either of these.

We are happy with your collaboration we'll figure it out, thanks for your efforts thus far!

**Comment by kaikreuzer** [comment]
> @kaikreuzer can we see this as a minor change or is there another way to comply to the contributor guidelines?

As one of his commits is signed, we can accept it that way as we anyhow squash on merge.

File: addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -21,7 +21,10 @@
 				<options>
 					<option value="0">North America (915MHz)</option>
 					<option value="1">Europe (868MHz)</option>
-					<option value="2">Australia (915MHz)</option>
+					<option value="2">Australia</option>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml]: Why did you remove the "(915MHz)" here, while the frequency is still mentioned for NA and Europe?

File: addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -21,7 +21,10 @@
...
+					<option value="3">Fine Offset (915MHz)</option>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml]: These options do not seem to make much sense, at least they do not fit the description, which says that this parameter specifies the band. We now have multiple entries for 915MHz and multiple ones for 868MHz.
Furthermore, at least to me "Fine Offset" doesn't have any helpful meaning (and if somebody asked me what the difference between "Europe" and "Fine Offset" is, I would have to shrug...).
Maybe some more explanation in the documentation could help here :-)

Comment (kermitd) [inline addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml]: I think if you owned the device itself it probably makes more sense.  The device documentation says this:

"m0“ selects the USA band at 915 MHz, „m1“ selects the Europe band at 868 MHz and “m2” selects Australia frequency band. “m3” selects reception of Fine Offset WH24 sensor unit in 915 MHz band and “m4” in 868 MHz band.

I've added the un-documented NZ mode ("m5") but otherwise tried to stay consistent with the original device documentation, including removing the Aus frequency.

The issue is that I don't think this parameter is really just band.  I think that the Davis devices that this thing supports have both bands and frequency hopping protocols that depend on local frequency regulations.  So in Aus it might be in 915mhz but the frequency hopping will be different from NA.   Then it seems to support a specific device (Fine Offset) that works in the same bands on different protocols. 

I don't *know* this for sure though so thought best to stick to what the manufacturer says, even though it is a bit confusing.

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml]: If the labels are exactly the same as what is found in the manufacturer documentation, it should be fine. But can you confirm that it indeed occurs there with and without band information?

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml]: @kermitd Ping :-)

**Comment by kermitd** [comment]
@kaikreuzer Better this time I think.  Options are clear that frequency mode encapsulates device, region and frequency, both on README and on thing_types.  Also managed to locate the NZ Davis frequency in other Davis documentation so have included this.
